Gaijinizer patch/Map160.txt

```diff
@@ -193,17 +195,17 @@ It's nothing...#Triketos
 
 　　　　　　海底列車の点検車両マニュアル
 > CONTEXT: Map160/events/6/pages/0/2/Dialogue
-Submarine train inspection vehicle manual#MTLed
+Undersea Train Maintenance Vehicle Manual#Ron
 > END STRING
 
 > BEGIN STRING
 島国だけでなく、海上施設の中継ステーションがある
 
  この海底列車は、おそらく世界最長です。
 > CONTEXT: Map160/events/6/pages/0/6/Dialogue
-As well as island countries, there are relay stations for
-offshore facilities. This undersea train is probably the
-longest in the world.#MTLed
+This undersea railway, with relay stations not just across
+island nations, but also built with maritime facilities on
+the ocean surface, is probably the longest in the world.#Ron
 > END STRING
```

Comment:
Let's rephrase this:
>_**This undersea relay, with relay stations built across island
nations and maritime facilities on the ocean's surface, is
one of the largest in the world.**_